src/pal/src/exception/seh.cpp

```diff
@@ -30,18 +28,19 @@ Module Name:
 #include "pal/init.h"
 #include "pal/process.h"
 #include "pal/malloc.hpp"
+#include "signal.hpp"
 
 #if HAVE_ALLOCA_H
-#include "alloca.h"
+#  include "alloca.h"
```

Comment:
I used it in https://github.com/dotnet/coreclr/pull/1687/files. The advantage of indented convention is it makes it easier to read. Will revert it. :)